docs/content/users/usage/commands.md

```diff
@@ -233,13 +233,13 @@ ddev config global --omit-containers=dba,ddev-ssh-agent
 * `--letsencrypt-email`: Email associated with Let’s Encrypt; `ddev global --letsencrypt-email=me@example.com`.
 * `--mutagen-enabled`: If `true`, web container will use Mutagen caching/asynchronous updates.
 * `--nfs-mount-enabled`: Enable NFS mounting on all projects globally.
-* `--no-bind-mounts`: If `true`, don’t use bind-mounts. Useful for environments like remote Docker where bind-mounts are impossible. (default `true`)
+* `--no-bind-mounts`: If `true`, don’t use bind-mounts. Useful for environments like remote Docker where bind-mounts are impossible. (default is equal to `--no-bind-mounts=true`)
```

Comment:
I don't know, maybe something like this will be clear.

Reply:
I thought “(default `true`)”—ignoring the typo you fixed—was clear, but maybe this is better? If you think so great, I don’t think this makes it any _less_ clear.